Integrations/integration-HybridAnalysis.yml

```diff
@@ -354,6 +363,186 @@ script:
 
     }
 
+
+    function quick_scan_result_to_entry(response, indicator, threshold, headers) {
+
+        var unknown_counter = 0 ;
+        var malicious_counter = 0 ;
+        //var num_elements = 0 ;
+        var Score = 0;
+
+        // create scanners entry
+        var scanners = [];
+        response.scanners.forEach(function(key) {
+            scanners.push(
+                {
+                    'Name': key.name,
+                    'Status': key.status,
+                    'Positives': key.positives
+                });
+            if (key.status == 'unknown' || key.status =='no-result' )
+                unknown_counter++;
+            else if (key.status == 'malicious')
+                malicious_counter++;
+       //     num_elements++;
+        });
+
+        //calculating DBot Score
+        if (unknown_counter == response.scanners.lenght)
+            Score = 0;
+        if (malicious_counter == 0)
+            Score = 1;
+        if (malicious_counter == 1)
+            Score = 2;
+        if (malicious_counter >= threshold)
+            Score = 3;
+
+        var Dbot_context = {
+            'Indicator': response.sha256,
+            'Vendor': "Hybrid Analysis",
+            'Score': Score,
+            }
+        var whitelist = {
+            'ID': response.whitelist[0].id,
+            'Value' : response.whitelist[0].value
```

Comment:
please split this to several lines for readability and clarity.
For example:
whitelist = response.whitelist
first_whitelisted_file = whitelist[0]
then the ID and Value.
However, this note might change after I understand the variable's meaning better